nixos/modules/misc/ids.nix

```diff
@@ -340,6 +340,7 @@
       cockroachdb = 313;
       zoneminder = 314;
       paperless = 315;
+      arangodb = 316;
```

Comment:
Does the service actually require a predefined `uid` and `gid`? Often times you can get away without.

Reply:
Yeah chance is very high that this isn't needed, cc https://github.com/NixOS/nixpkgs/pull/65698